index.html

```diff
@@ -2,315 +2,523 @@
-            </li>
-          </ul>
+          </div>
         </nav>
 
-        <!-- Page Content  -->
-        <div id="content">
-
-          <div class="subcontent active" id="home">
-                <nav class="navbar navbar-expand-lg navbar-light bg-light">
-                    <div class="container-fluid">
-                        <button type="button"  class="btn btn-info sidebarCollapse">
-                            <i class="fas fa-align-left"></i>
-                        </button>
-
-                    </div>
-                </nav>
-              <div class="container home">
-                            <div class="container1">
-                                  <div class="main">
-                                    <div class="heading"> About </div>
-                                    <hr>
-                                    <div id="info">
-                                      <p> Hey Folks!
-                                This is an initiative by the students of 4th year to build a coding club that will serve the purpose of providing the environment for coders to flourish and shine. We will create a platform which allows students to gain assistance and mentorship to enhance their coding ability. Our aim is to propagate the enthusiasm for coding in the institute and especially amongst freshmen. Moreover, the best skilled coders will be there to guide you through the process. This endeavour is not just for creating a culture for competitive coding but we also will mentor you in technologies like Machine Learning , Web Development , Android App Development and many more. You may fill the field of your interest in the form.
-
-                                So come forward and be a part of CodeShows family.</p>
-                                    </div>
-
-                                    <div class="clear"></div>
-                                  </div>
-
-
-                      </div>
+        <div class="container contest">
+          <div class="container1">
+            <div class="main">
+              <div class="heading">CONTEST LINKS</div>
+              <div class="con">
+                <div class="contest1" onclick="showans('ans1')">
+                  <i class="fas fa-trophy"></i>
+                  Contest 5: (Code Friday) (8 March 2019)
+                  <i class="fas fa-sort-down" style="float:right"></i>
+                </div>
+                <p id="ans1"><a href="http://hck.re/CFFC2" target="_blank">Contest 5-Combined for all </a>
+                </p>
               </div>
-
+              <div class="con">
+                <div class="contest1" onclick="showans('ans2')">
+                  <i class="fas fa-trophy"></i>
+                  Contest 4: (24 Feb 2019)
+                  <i class="fas fa-sort-down" style="float:right"></i>
+                </div>
+                <p id="ans2"><a href="http://hck.re/td11545" target="_blank">Contest 4-Combined for all</a>
+                </p>
```

Comment:
Do a few updates and then I can merge:
1. Add some padding inside the collapsible body of the contests
2. Remove the rounded borders from the contest accordian

Reply:
So you want to the borders to be entirely rectangular? I have this image with the rectangular borders and the padding. 

![2019-04-16-134521_1920x1080_scrot](https://user-images.githubusercontent.com/36107758/56236290-db089880-604e-11e9-8023-65a3436e13af.png)
![2019-04-16-134532_1920x1080_scrot](https://user-images.githubusercontent.com/36107758/56236291-db089880-604e-11e9-9ab7-fdc2b9fe7669.png)

Reply:
This one looks better